stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -1054,17 +1054,32 @@ function(_compile_swift_files
     set(copy_legacy_layouts_dep)
   endif()
 
+  # Produce a file that applies all Fix-Its, as a convenience.
```

Comment:
It is not entirely clear to me from the commit message and the comments if the logic to emit the fixit will be dropped from this PR before merging, or if it will stay for future adoptions.

In the latter case,
* would it be possible to port this logic to the new build systems as well, so to maintain parity?
* could we gate this under a CMake `option`, so it is enabled at desk only when needed and we avoid landing those in packaged toolchains and SDKs?

Reply:
I think it's best if I drop this before merging. It's a tool to get us to enable strict memory safety throughout the standard libraries, but it's probably a bad idea to keep around.